### DISCUSSION TITLE
看到这个没绷住

### DISCUSSION THREAD

**Discussion by YenalyLiew** [xiaojieonly/Ehviewer_CN_SXJ#1137] [General]
![Screenshot_2024-04-12-12-57-14-688_com.github.android-edit.jpg](https://github.com/xiaojieonly/Ehviewer_CN_SXJ/assets/92662107/a5cbe9d0-ab38-4762-9155-89cebefd4d80)

**Comment by hfhxghc** [comment]
安装链接在哪里呀？

**Comment by xiaojieonly** [comment]
被自己写的弱智代码气到了
